rotkehlchen/premium/premium.py

```diff
@@ -476,6 +510,24 @@ def query_premium_components(self) -> str:
 
         return result['data']
 
+    def fetch_limits(self) -> UserLimits:
+        if self._cached_limits is not None:
+            return self._cached_limits
+
+        try:
+            response = self.session.get(
+                f'{self.rotki_nest}{(method := "limits")}',
+                params=self.sign(method=method),
+                timeout=ROTKEHLCHEN_SERVER_TIMEOUT,
+            )
+        except requests.exceptions.RequestException as e:
+            msg = f'Could not connect to rotki server due to {e!s}'
+            log.error(msg)
+            raise RemoteError(msg) from e
+
+        self._cached_limits = cast('UserLimits', _process_dict_response(response))
```

Comment:
i would add some kind of log here because otherwise is hard to debug issues with the tier limits